jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
+1 to @knorth55 
This is configuration for pick task. `data` is too abstract, I think.